pkg/kv/bulk/sst_batcher.go

```diff
@@ -591,7 +558,13 @@ func (b *SSTBatcher) addSSTable(
 				beforeSend := timeutil.Now()
 				br, pErr := b.db.NonTransactionalSender().Send(ctx, ba)
 				sendTime := timeutil.Since(beforeSend)
-				b.flushCounts.sendWait += sendTime
+				b.stats.sendWait += sendTime
+
+				if br != nil && len(br.BatchResponse_Header.RangeInfos) > 0 {
+					for i := range br.BatchResponse_Header.RangeInfos {
+						b.stats.sendWaitByStore[br.BatchResponse_Header.RangeInfos[i].Lease.Replica.StoreID] += sendTime
```

Comment:
> Do you need concurrency control here? 

No, this is all single threaded.

> only ever be one iteration in this loop. ... which is probably true, right?

Yep, should be, the request is unsplitable. 

> maybe we should be dividing by the length?

I dunno, if it is somehow >1 -- so already an edge case here -- and the amount trivial then it's not material in the bigger picture to over-attribute the sum to every store. On the other hand, if the amount is non-trivial, like say, minute, my _guess_ is that usually that's down to a single slow store, and in that case, dividing it around could help it "hide" its slowness behind its peers, which is exactly that this debugging logging is trying to suss out. I think in that case I'd rather just say "one or more of all of these took a long time" and ding all associated stores the full amount, since they were all involved in my waiting that duration.